DEPS

```diff
@@ -459,7 +459,7 @@ deps = {
      'packages': [
        {
         'package': 'flutter/android/sdk/platforms',
-        'version': 'version:28r6'
+        'version': 'version:29r1'
```

Comment:
How sure are we that this is safe?  This means that now all our builds will go against the 29 SDK.  Is it stable enough yet?

Reply:
As mentioned in the [buildroot PR](https://github.com/flutter/buildroot/pull/291) (posting here for posterity as well), this isn't certain, but it'd be ideal to support system gestural navigation as soon as possible so that Flutter devs can catch up to this update. Originally, I was using reflection to handle this particular case, but a lint recommended against doing so. 

On one hand, we could move forward with using reflection, and once v29 of the Android SDK is stable, roll v29 into the engine and then remove the use of reflection.

Reply:
Ok. Can we roll th eother SDK packages with this one?  Especially build tools.